lang/lang.common.json

```diff
@@ -3,7 +3,11 @@
         "name": "My Bot",
         "author": "My Name"
     },
-    "commands": {
+    "contextCommands": {
+        "viewJoinDate": "View join date!",
+        "viewMessageSentDate": "View message sent date!"
+    },
```

Comment:
Can you put this section after chatCommands?

Reply:
Actually can we separate this into "userCommands" and "messageCommands" and put below the "chatCommands"?